sourcegraph/README.md

```diff
@@ -0,0 +1,75 @@
+# Sourcegraph Datadog Services Map Extension
+
+## Overview
+
+![Datadog Services Map Screenshot](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sourcegraph/images/sourcegraph-datadog-services-map-extension.png)
+
+The Datadog Services Map Sourcegraph Extension brings the context of Datadog APM directly into your code on Sourcegraph and on your code host through [the Sourcegraph Browser Extension][1]. 
+
+* See which services call or are called by other services.
+* Go directly to the [Datadog APM Services Map][2] for your service.
+
+## Setup
+
+### 1. Enable the extension 
+
+Visit the Sourcegraph extensions registry at yourSourcegraphInstance.com/extensions and search for `datadog-service-map`. Enable the extension. 
+
+Sourcegraph Cloud users can directly enable the extension at https://sourcegraph.com/extensions/sourcegraph/datadog-service-map. 
+
+### 2. Add configuration keys to the preferred Sourcegraph settings level
+
+To enable this extension for everyone on your Sourcegraph instance, add these to your global settings `/site-admin/global-settings`. 
+
+To enable it for just users in your organization, add this to your organization settings `/organizations/orgName/settings`. 
+
+To enable it just for yourself, add these to your user settings `/user/username/settings`. 
+
+```json
+"datadogServiceMap.apiKey": "DD_API_KEY",
+"datadogServiceMap.applicationKey": "DD_APPLICATION_KEY",
+"datadogServiceMap.env": "DD_SERVICE_MAP_ENVIRONMENT",
+"datadogServiceMap.corsAnywhereUrl": "CORS_ANYWHERE_URL",
+```
+
+If you don't have a CORS anywhere URL, you can use the Sourcegraph demo: `https://sourcegraph-demo-cors-anywhere.herokuapp.com`
```

Comment:
nit. We could define `https://sourcegraph-demo-cors-anywhere.herokuapp.com` as a link